### DISCUSSION TITLE
Issue setting up custom SMTP with Mailgun?

### DISCUSSION THREAD

**Discussion by jimmiejackson414** [supabase/supabase#7444] [Questions] [answered]
I've created a Mailgun account in an attempt to hook it up to Supabase as a custom SMTP provider. When a user goes to sign up however, I am receiving the following network error:
```
{"code":500,"msg":"Error sending confirmation mail","error_id":"bb29c6f4-4d99-4c85-8772-5f2fb042221c"}
```

When I take a look at the Auth Logs, I see this:
```
time="2022-06-26T22:10:34Z" level=error msg="500: Error sending confirmation mail" component=api error="Error sending confirmation email: dial tcp 34.83.33.178:25: i/o timeout" method=POST path=/signup referer="https://www.mydomain.io/" remote_addr="24.19.220.31:22601" request_id=bb29c6f4-4d99-4c85-8772-5f2fb042221c
```

I've verified that my credentials are correct, so I'm not entirely sure what could be the issue here. Any ideas?

**Comment by jimmiejackson414** [comment, marked as answer]
Finally figured it out, posting the solution in case anyone else runs in to this issue.

The Domain Settings from Mailgun are what confused me, I switched to port 587 as recommended in the docs, and then realized I had the "SMTP admin email" and "SMTP user" fields switched up. Changing those things fixed everything.

**Comment by andyjamesn** [comment]
Could you provide some direction on how you setup mailgun as a custom SMTP provider to Supabase?

**Reply by eddyizm** [reply]
It is a little confusing, especially if you haven't created a user yet but just was able to test it successfully.

**Comment by jimmiejackson414** [comment]
@andyjamesn I'm not sure what you've done so far, but within Mailgun I created a new domain. Then I went into Sending > Domain Settings > SMTP credentials and inserted those settings into my Supabase SMTP settings.
